00new_packages.autobuild

```diff
@@ -13,5 +13,5 @@ end
 
 cmake_package 'gui/rock_webapp'
 ruby_package 'tools/rest_api' do |pkg|
-    Autoproj.env_set 'ROCK_WEBAPP', File.join(pkg.prefix, 'share', 'rest_api')
+    Autoproj.env_set 'ROCK_WEBAPP', File.join(Autoproj.build_dir, 'share', 'rest_api')
```

Comment:
If you would read my comment more carefully, you would notice that I say that the setting should be set for **gui/rock_webapp**'s prefix, not **tools/rest_api**.

The very fact that taking the prefix of `tools/rest_api` does not work should have rang a bell already. It's not working because **it's not the right prefix**.